tests/infrared/16/stf-connectors.yaml.template

```diff
@@ -63,6 +63,12 @@ custom_templates:
             tripleo::profile::base::metrics::collectd::amqp_host: "%{hiera('storage')}"
             tripleo::profile::base::metrics::qdr::listener_addr: "%{hiera('storage')}"
 
+        NovaEnableRbdBackend: true
+        CinderEnableRbdBackend: true
+        CinderBackupBackend: ceph
+        GlanceBackend: rbd
+        GnocchiBackend: rbd
+
```

Comment:
I need to drop this in to a separate PR really because I'm not even sure it does exactly what I want. The idea is to make sure the Ceph environment is actually getting used for data storage.